gammapy/data/tests/test_event_list.py

```diff
@@ -19,6 +19,7 @@ def test_EventList():
     assert event_list.radec[0].to_string() == '82.7068 19.8186'
     assert event_list.galactic[0].to_string() == '185.956 -7.69277'
     assert event_list.altaz[0].to_string() == '46.2059 31.2001'
+    assert_allclose(event_list.offset[0].value, 1.904497742652893)
```

Comment:
This is failing on travis-ci:
https://travis-ci.org/gammapy/gammapy/jobs/106254928#L1511
```
>       assert_allclose(event_list.offset[0].value, 1.904497742652893)
E       AssertionError: 
E       Not equal to tolerance rtol=1e-07, atol=0
E       
E       (mismatch 100.0%)
E        x: array(1.904497504234314)
E        y: array(1.904497742652893)
```

1e-7 is at the level of 32-bit precision.
You have to reduce the tolerance of the assert a little bit to make this test pass in a robust way.
My suggestion in this case would be that you do this:
```
assert_allclose(event_list.offset[0].value, 1.904497742652893, rtol=1e-5)
```